.github/workflows/lra-profile.yml

```diff
@@ -0,0 +1,49 @@
+name: LRA-CORE
+
+on:
+  push:
```

Comment:
Might as well add workflow_dispatch too?

Reply:
Good idea, updated it.